static/templates/settings/notification-settings.handlebars

```diff
@@ -35,7 +35,7 @@
             {{/each}}
         </div>
 
-        <div id="other_notifications">
+        <div id="other_notifications" class="m-10 inline-block">
```

Comment:
What's the story with this change?

Reply:
It was required to limit the width of the input field, so it matched other fields.

Reply:
I see, because the label of one of the new options is long?  Makes sense.